docs/code_style.md

```diff
@@ -46,11 +41,9 @@ pip install -e ".[lint,mypy]"
     Auto-fix imports with:
 
     ```sh
-    isort -rc synapse tests
+    isort .
     ```
 
-    `-rc` means to recursively search the given directories.
-
```

Comment:
is `-rc` no longer needed?

Reply:
Correct. Since isort 5.0.0 (July 2020) the `--recursive` option is removed and directories are sorted recursively by default.